week-8/Homework/mandatory/4-programmer-humour/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Programmer Jokes</title>
+  </head>
+  <body>
+    <div id="container" class="center">
+      <h1 id="title"></h1>
+      <img id="joke" src="" alt="" />
+    </div>
+    <script src="script.js"></script>
```

Comment:
The page looks a bit puritan, but it does exactly what was asked to be done.

Reply:
I have very bad taste when it comes to choosing colors and I totally lack creativity in design